wolox-books/src/styles.scss

```diff
@@ -1 +1,97 @@
-/* You can add global styles to this file, and also import other style files */
+$equalizer-black: #000;
```

Comment:
create a file called "equalizer.scss" and then impor it here. We will have more global classes later on